src/viam/sdk/resource/resource_manager.cpp

```diff
@@ -65,36 +65,35 @@ std::string get_shortcut_name(const std::string& name) {
     return name_split.at(name_split.size() - 1);
 }
 
-void ResourceManager::do_add(const Name& name, const std::shared_ptr<Resource>& resource) {
+void ResourceManager::do_add(Name name, std::shared_ptr<Resource> resource) {
```

Comment:
This could be `Name const& name, std::shared_ptr<Resource> resource)` because the first parameter is only observed, but the second one is actually moved.